models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -23,6 +23,28 @@
 ##' @export
 ##' @author Shawn Serbin, Michael Dietze
 model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date, delete.raw, revision, overwrite = FALSE) {
+  
+  sipnet2datetime <- function(sipnet_tval, base_year, base_month = 1,
```

Comment:
I would stick this outside this function, and possibly `@export` it (no need to thoroughly document yet -- we can punt on that). No reason this function can't stand alone, and making it so will make it usable to others and accessible to unit tests.

Reply:
OKie dokie

Reply:
I  realized this may not work for forecast runs if the met/run does not start at time 0.  That is, what if we start June 1 ? need to think of a way to tell model2netCDF.SIPNET whether or not the met is CF compliant.  I guess pass force_cf=TRUE to sipnet2datetime?